pandas-stubs/io/formats/style.pyi

```diff
@@ -278,7 +278,7 @@ class Styler(StylerRenderer[Styler]):
     ) -> Styler: ...
     def highlight_null(
         self,
-        null_color: str = ...,
+        color: str | None = ...,
```

Comment:
docs are missing `None` as an argument - can you create an issue for that?

Reply:
Docs on that one are pretty wrong.  Have wrong color too